src/misc/crypt/crypt_ltc_mp_descriptor.c

```diff
@@ -8,7 +8,10 @@
  */
 #include "tomcrypt.h"
 
-ltc_math_descriptor ltc_mp;
+/* Initialize ltc_mp to nulls, to force allocation on all platforms, including macOS. */
+ltc_math_descriptor ltc_mp = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
it should be enough to use just:
```C
ltc_math_descriptor ltc_mp = { 0 };
```

Reply:
Doing so produces a warning. Do you care? The rest of the code is pretty warning-clean.

```
src/misc/crypt/crypt_ltc_mp_descriptor.c:12:34: warning: missing field 'bits_per_digit' initializer [-Wmissing-field-initializers]
ltc_math_descriptor ltc_mp = { 0 };
                                 ^
1 warning generated.
```

Reply:
Hmm, warning is not good

Reply:
What about `ltc_math_descriptor ltc_mp = { };` ?

Reply:
Doing that works for me, and produces no warning, but from what I've read might only be a C++ or >C99 thing; I don't know if it would work under other compilers. I can test it out if I have a chance tomorrow. (It's late night here and I'm about to go to bed.)